discogs-labels.csv

```diff
@@ -308,3 +308,4 @@ Label ID, Label Name, Link
 839564, LoveStyle Records, https://www.discogs.com/label/839564-LoveStyle-Records
 1429, Life Enhancing Audio, https://www.discogs.com/label/1429-Life-Enhancing-Audio
 26064, N.E.W.S., https://www.discogs.com/label/26064-NEWS
+662216, Dataflow UK, https://www.discogs.com/label/662216-Dataflow-UK
```

Comment:
is using this one better? https://www.discogs.com/label/3736-Dataflow-Music
it has more releases and a picture =)

Reply:
Done

Reply:
thanks!